api/app/attendance/api.py

```diff
@@ -117,35 +117,23 @@ def post(self):
         # Other Fields
         unavilable_response = None
         invitedguest_role = unavilable_response
-        is_guest_registration = unavilable_response
+        is_invited_guest = unavilable_response
         has_accepted_accom_award = unavilable_response
 
-        registration = None

```

Comment:
This is going to get tricky actually. 
Because of all the permutations. 

@KaleabTessera  @avishkar58 
How does anyone feel about a raw sql query for all this? The one Avishkar sent earlier?